### PR TITLE
Fix javadoc in NoSQLHandleConfig

### PR DESCRIPTION
**Pull request by gmfeinberg**
Fix javadoc in NoSQLHandleConfig, modify format in CHANGELOG.md to conform to keepachangelog.

### CODE REVIEW

File: driver/src/main/java/oracle/nosql/driver/NoSQLHandleConfig.java [hunk 1]
```diff
@@ -581,8 +581,8 @@ public NoSQLHandleConfig setRequestTimeout(int timeout) {
     }
 
     /**
-     * Sets the default table request timeout. The default timeout is
-     * 5 seconds. The table request timeout can be specified independently
+     * Sets the default table request timeout.
+     * The table request timeout can be specified independently
      * of that specified by {@link #setRequestTimeout} because table requests can
      * take longer and justify longer timeouts. The default timeout is 10
      * seconds (10000 milliseconds).
```
Comment (lindaqlee) [inline driver/src/main/java/oracle/nosql/driver/NoSQLHandleConfig.java:588]: Am I reading this diff right? Shouldn't we be taking out the line about the default timeout being 10 seconds -- isn't it 5 seconds?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## Unpublished
+## [Unreleased]
+
 ### Added
 - Cloud only:
   - Added new new SignatureProvider constructors to allow use of an instance principal with delegation token in a file for authorization and authentication.
```
Comment (lindaqlee) [inline CHANGELOG.md:10]: Typo: (new new)

File: driver/src/main/java/oracle/nosql/driver/NoSQLHandleConfig.java (hunk 1 above)
Comment (lindaqlee) [reply driver/src/main/java/oracle/nosql/driver/NoSQLHandleConfig.java:588]: Or is it 5 for regular requests and 10 seconds for table requests?

Comment (gmfeinberg) [reply driver/src/main/java/oracle/nosql/driver/NoSQLHandleConfig.java:588]: Correct -- 5 for regular requests and that is documented. 10 for table requests.

**Pull request merged**